liquidctl/driver/kraken_three_x.py

```diff
@@ -0,0 +1,327 @@
+    'moving-alternating-4':                 (0x05, 0x04, 4, 1, 2),
+    'moving-alternating-5':                 (0x05, 0x05, 4, 1, 2),
+    'moving-alternating-6':                 (0x05, 0x06, 4, 1, 2),
+    'backwards-moving-alternating-3':       (0x05, 0x03, 4, 1, 2),
+    'backwards-moving-alternating-4':       (0x05, 0x04, 4, 1, 2),
+    'backwards-moving-alternating-5':       (0x05, 0x05, 4, 1, 2),
+    'backwards-moving-alternating-6':       (0x05, 0x06, 4, 1, 2),
+    'pulse':                                (0x06, 0x00, 5, 1, 8),
+    'breathing':                            (0x07, 0x00, 6, 1, 8),
+    'super-breathing':                      (0x03, 0x00,10, 1, 8),
+    'candle':                               (0x08, 0x00, 0, 1, 1),
+    'starry-night':                         (0x09, 0x00, 5, 1, 1),
+    'rainbow-flow':                         (0x0b, 0x00, 2, 0, 0),
+    'super-rainbow':                        (0x0c, 0x00, 2, 0, 0),
+    'rainbow-pulse':                        (0x0d, 0x00, 2, 0, 0),
+    'backwards-rainbow-flow':               (0x0b, 0x00, 2, 0, 0),
+    'backwards-super-rainbow':              (0x0c, 0x00, 2, 0, 0),
+    'backwards-rainbow-pulse':              (0x0b, 0x00, 2, 0, 0),
+    'loading':                              (0x10, 0x00, 8, 1, 1),
+    'tai-chi':                              (0x0e, 0x00, 7, 1, 2),
+    'water-cooler':                         (0x0f, 0x00, 6, 2, 2),
+    'wings':                                (None, 0x00,11, 1, 1),
+}
+_STATIC_VALUE = {
+    0x02: 0x08,
+    0x04: 0x01
+}
+_SPEED_VALUE = {
+    # (slowest, slow, normal, fast, fastest)
+    0: ([0x32, 0x00], [0x32, 0x00], [0x32, 0x00], [0x32, 0x00], [0x32, 0x00]),
+    1: ([0x50, 0x00], [0x3c, 0x00], [0x28, 0x00], [0x14, 0x00], [0x0a, 0x00]),
+    2: ([0x5e, 0x01], [0x2c, 0x01], [0xfa, 0x00], [0x96, 0x00], [0x50, 0x00]),
+    3: ([0x40, 0x06], [0x14, 0x05], [0xe8, 0x03], [0x20, 0x03], [0x58, 0x02]),
+    4: ([0x20, 0x03], [0xbc, 0x02], [0xf4, 0x01], [0x90, 0x01], [0x2c, 0x01]),
+    5: ([0x19, 0x00], [0x14, 0x00], [0x0f, 0x00], [0x07, 0x00], [0x04, 0x00]),
+    6: ([0x28, 0x00], [0x1e, 0x00], [0x14, 0x00], [0x0a, 0x00], [0x04, 0x00]),
+    7: ([0x32, 0x00], [0x28, 0x00], [0x1e, 0x00], [0x14, 0x00], [0x0a, 0x00]),
+    8: ([0x14, 0x00], [0x14, 0x00], [0x14, 0x00], [0x14, 0x00], [0x14, 0x00]),
+    9: ([0x00, 0x00], [0x00, 0x00], [0x00, 0x00], [0x00, 0x00], [0x00, 0x00]),
+    10:([0x37, 0x00], [0x28, 0x00], [0x19, 0x00], [0x0a, 0x00], [0x00, 0x00]),
+    11:([0x6e, 0x00], [0x53, 0x00], [0x39, 0x00], [0x2e, 0x00], [0x20, 0x00]),
+}
+_ANIMATION_SPEEDS = {
+    'slowest': 0x0,
+    'slower': 0x1,
+    'normal': 0x2,
+    'faster': 0x3,
+    'fastest': 0x4,
+}
+_ACCESSORY_NAMES = {
+    0x01: "HUE+ LED Strip",
+    0x02: "AER RGB 1",
+    0x04: "HUE 2 LED Strip 300 mm",
+    0x05: "HUE 2 LED Strip 250 mm",
+    0x06: "HUE 2 LED Strip 200 mm",
+    0x08: "HUE 2 Cable Comb",
+    0x0a: "HUE 2 Underglow 200 mm",
```

Comment:
Missing a recent addition to the SDV2 driver:

```
0x09: "HUE 2 Underglow 300 mm",
```

(I'll fix this small duplication later)

Reply:
17d4c63